Client/Frontend/Browser/HomePanel/CustomTheme.swift

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
FYI, that copyright statement is missing the following (as the first line):
> Copyright 2020 The Brave Authors. All rights reserved.

Reply:
We use this copy across all files, do you mind creating a ticket for it?
Better to update it in separate PR

Reply:
https://github.com/brave/brave-ios/issues/2441